components/renku_data_services/utils/core.py

```diff
@@ -12,3 +15,22 @@ def get_ssl_context():
     if custom_cert_file:
         context.load_verify_locations(cafile=custom_cert_file)
     return context
+
+
+def merge_api_specs(*args):
+    """Merges API spec files into a single one."""
+    merger = Merger(
+        type_strategies=[(list, "append_unique"), (dict, "merge"), (set, "union")],
```

Comment:
is `append_unique` a built-in strategy?

Reply:
Yes -> https://github.com/toumorokoshi/deepmerge/blob/v1.1.0/deepmerge/strategy/list.py

Reply:
I guess their docs need an update 😆

Reply:
I didn't know they had docs :rofl: I just Ctrl+Click in vscode.

Reply:
obvs the better approach 😆 🙈